README.md

```diff
@@ -0,0 +1,14 @@
+# IcedTeaWeb
+This repository contains all sources of IcedTeaWeb. 
+
+## About IcedTeaWeb
+IcedTeaWeb is an open source implementation of JSR-56 that is better know as WebStart.
+
+## Development origin
+The project was former hosted as part of the classpath.org project but moved to github.
```

Comment:
"formerly" instead of "former"?

Should we add the link to the web site?: https://icedtea.classpath.org